amf-shapes/shared/src/main/scala/amf/shapes/internal/spec/oas/parser/InlineOasTypeParser.scala

```diff
@@ -101,9 +101,9 @@ case class InlineOasTypeParser(
   protected def isOas3: Boolean = version.isInstanceOf[OAS30SchemaVersion]
 
   protected def moveExamplesToUnion(parsed: AnyShape, union: UnionShape): Unit = {
-    val AmfArray(values, fieldAnnotations) = parsed.fields.get(ExamplesField.Examples)
+    val AmfArray(values, annotations) = parsed.fields.get(ExamplesField.Examples)
     if (values.nonEmpty) {
-      union.setWithoutId(ExamplesField.Examples, AmfArray(values), fieldAnnotations)
+      union.setWithoutId(ExamplesField.Examples, AmfArray(values, annotations), annotations)
```

Comment:
shouldn't this be syntethized, virtual or something like that?

Reply:
it should actually be inferred, because it's content it's explicit in the document, just not exactly in an union